### PR TITLE
releasenotes: add note about CentOS 8 EOL in Dec 2021

### PR DESCRIPTION
**Pull request by rhtyd**
This adds notes in release notes's compatibilty matrix page and about page.

### CODE REVIEW

**Comment by rhtyd** [comment]
requesting docbuild

**Review by svenvogel — APPROVED** [review]

File: source/releasenotes/compat.rst [hunk 1]
```diff
@@ -22,8 +22,8 @@ Supported OS Versions for Management Server
 This section lists the operating systems that are supported for running
 CloudStack Management Server.
 
--  Ubuntu 16.04 LTS, 18.04, 20.04 LTS
--  CentOS versions 7.x, 8.x
+-  Ubuntu 16.04 LTS, 18.04 LTS, 20.04 LTS
```
Comment (GabrielBrascher) [inline source/releasenotes/compat.rst:25]: Ubuntu 16.04 also has an EOL coming soon, in April 2021. Should add such a disclaimer for it as well?

**Review by GabrielBrascher — APPROVED** [review]
+1 on the proposed changes; I just pointed out Ubuntu 16.04 EOL to be considered as well.

**Review by weizhouapache — APPROVED** [review]

File: source/releasenotes/compat.rst (hunk 1 above)
Comment (rhtyd) [reply source/releasenotes/compat.rst:25]: @GabrielBrascher it's a normal case where the EOL date hasn't changed and I suppose most users would be aware that 16.04 is old enough by now.

**Pull request merged**